client/js/components/EnvironmentVariables/EnvironmentVariables.less

```diff
@@ -0,0 +1,22 @@
+// EnvironmentVariables.less
+@import "../../../less/variables";
+
+.environmentVariables {
+	overflow-y: scroll !important;
+}
+
+.environmentVariables-Notice--error {
+	background-color: @color-error-bg !important;
```

Comment:
@ifandelse I see you are defining them in the variables file, this is fine for now and we can decide if its worth moving later.